intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -556,7 +556,7 @@ \subsection{Getting cluster state}
 
 
 \begin{prompt}
-%\shellcmd{module swap cluster/phanpy}%
+%\shellcmd{module swap cluster/kirlia}%
```

Comment:
If you change this, also change the output below...

Reply:
done